j2lint/linter/indenter/node.py

```diff
@@ -135,8 +135,11 @@ def check_indentation(self, result, lines, line_no=0, indent_level=0):
                         self.check_indent_level(result, node)
                         jinja_node_stack.pop()
                         return line_no
-                    raise JinjaLinterError(
-                        "Tag is out of order '{}'".format(node.tag))
+                message = "Tag is out of order '{}'".format(node.tag)
+                error = self.create_indentation_error(node, message)
+                result.append(error)
+                raise JinjaLinterError(
```

Comment:
Can you add similar message/error and append to result for other cases where we raise JinjaLinterError?